.github/workflows/cypress-test.yml

```diff
@@ -44,34 +45,42 @@ jobs:
           plugin-name: ${{ env.PLUGIN_NAME }}
           plugin-version: ${{ env.PLUGIN_VERSION }}
 
-      - name: Run Opensearch with A Single Plugin
-        uses: opensearch-project/security/.github/actions/start-opensearch-with-one-plugin@main
+      - name: Run Opensearch with security
+        uses: derek-ho/start-opensearch@v2
         with:
           opensearch-version: ${{ env.OPENSEARCH_VERSION }}
-          plugin-name: ${{ env.PLUGIN_NAME }}
-          setup-script-name: setup
-          admin-password: myStrongPassword123!
+          plugins: "file:$(pwd)/${{ env.PLUGIN_NAME }}.zip"
+          security-enabled: true
+          admin-password: ${{ env.OPENSEARCH_INITIAL_ADMIN_PASSWORD }}
+
+      # Configure the Dashboard
+      - name: Configure OpenSearch Dashboards for cypress
+        run: |
+          cat << 'EOT' > cypress-config.yml
+          server.host: "0.0.0.0"
+          opensearch.hosts: ["https://localhost:9200"]
+          opensearch.ssl.verificationMode: none
+          opensearch.username: "kibanaserver"
+          opensearch.password: "kibanaserver"
+          opensearch.requestHeadersWhitelist: [ authorization,securitytenant ]
+          opensearch_security.multitenancy.enabled: true
+          opensearch_security.multitenancy.tenants.preferred: ["Private", "Global"]
+          opensearch_security.readonly_mode.roles: ["kibana_read_only"]
+          opensearch_security.cookie.secure: false
+          opensearch_security.multitenancy.enable_aggregation_view: true
+          EOT
+          echo 'HERE IS THE DASHBOARD CONFIG FILE: '
+          cat cypress-config.yml
```

Comment:
nit: If this is the dashboards config file, I think it should be named accordingly.

Reply:
also, why is the file being `cat` here?

Reply:
Cat is just to show what the file is to the user when running the step. @cwperks I can rename it if you feel strongly but I think it's clear from the step name what it is doing, what do you think?

Reply:
do we need the cat?

Reply:
With the cypress config file called `cypress.config.js` I think its confusing at first glance that a file named `cypress-config.yml` is the file used to configure opensearch dashboards.